docs/install/prerequisites.md

```diff
@@ -1,11 +1,18 @@
 # Prerequisites
 
+Portable (CO:RE) option:
 - Linux kernel version >= 4.18
-- Linux kernel headers available under conventional location (see [Linux Headers](../headers) section for more info)
+- `CONFIG_DEBUG_INFO_BTF` Linux compile option enabled (only needed for running, not compilation)
```

Comment:
IMO the kernel config is not the requisite, BTF is. I suggest something similar to my other comment about this